backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/dao/impl/LpadQueryFilterModifier.java

```diff
@@ -91,6 +95,6 @@ public Object convertValue(final String columnName, final Object value, final Ob
 
 		final String str = (String)value;
 
-		return StringUtils.leftPad(str.trim(), size, padStr);
+		return StringUtils.leftPad(str.trim(), size, padStr).subSequence(0,size);
```

Comment:
pls document that we do subSequence because we want to have the same behaviour like Postgresql's LPAD.

same for the RPAD.

Reply:
done